Description: don't show notification if not upgrading from a previous version
Author: Marc Deslauriers <marc.deslauriers@ubuntu.com>
Bug: http://code.google.com/p/ibus/issues/detail?id=1677
Bug-Ubuntu: https://bugs.launchpad.net/ibus/+bug/1255542
Forwarded: yes

Index: ibus-1.5.14/ui/gtk3/panel.vala
===================================================================
--- ibus-1.5.14.orig/ui/gtk3/panel.vala
+++ ibus-1.5.14/ui/gtk3/panel.vala
@@ -758,6 +758,7 @@ class Panel : IBus.PanelService {
         string current_version = null;
 
         if (compare_versions(prev_version, "1.5.3") < 0)
+	if (prev_version != "" && compare_versions(prev_version, "1.5.3") < 0)
             update_version_1_5_3();
 
         if (compare_versions(prev_version, "1.5.8") < 0)
